packages/blade/src/components/Divider/Divider.stories.tsx

```diff
@@ -133,13 +135,17 @@ const DividerVerticalTemplate: ComponentStory<typeof DividerComponent> = () => {
           flexDirection="row"
           height="50px"
         >
-          <Button variant="secondary" marginRight="spacing.4" isFullWidth>
-            Sign up
-          </Button>
+          <BaseBox display="flex" flex={1}>
```

Comment:
is this a related change?

Reply:
Not really but story was breaking on RN so just fixed it